src/main/java/com/practice/test/IncomeTaxCalculatorService.java

```diff
@@ -3,6 +3,11 @@
 public interface IncomeTaxCalculatorService {
 
     float incomeTaxCalculation(float income);
+    float SB = .105f;
+    float S = .175f;
```

Comment:
Can you please move these constants to a Enum object, you can search enum's in java